packages/utils/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    extends: '../../.eslintrc.js',
+    parserOptions: {
+        project: './tsconfig.json',
+    },
+    rules: {
+        'prefer-object-spread': 'off', // prefer Object.assign
```

Comment:
why is this off? I don't think this good choice, we should go with most of JS community that using spread. There is also no any big advantage of using Object.assign instead https://stackoverflow.com/questions/32925460/object-spread-vs-object-assign

Reply:
if it's because `mergeObjects` then disable it just for that one file

Reply:
actually if you are writing npm library it's better not to have `__assign` polyfill added by typescript compiler to __each__ file in lib directory where spread is used (check transpiled packages/components/lib for example)

Reply:
I checked utxo build output and I am not sure if I understand, can you explain it little bit more? 

I think assign polyfill is used to polyfill `Object.assign` so it should be polyfilled everytime when `Object.assign` is so this rule won't help us at all. And spread operator is simply transpiled to `Object.assign` without any polyfill. I also checked our `targed` we are using `es6` so we are not using polyfill for assign at all.

Reply:
`@trezor/utxo-lib` and `@trezor/blockchain-link` are good examples since they are low-level libs like this `@trezor/utils`. They both have `prefer-object-spread` disabled and i gave special care not to have any polyfills in `lib` directory. 
Polyfills should be added only one time, on high-level builds like connect iframe or suite-web

In given example those 3 lines
https://github.com/trezor/trezor-suite/blob/develop/packages/rollout/src/index.ts#L245-L248
```
return {
        ...infoByBootloader,
        binary: modifyFirmware({ fw, features: parsedFeatures }),
};
```
are transpiled to `packages/rollout/lib/index.js`
```
var __assign = (this && this.__assign) || function () {
    __assign = Object.assign || function(t) {
        for (var s, i = 1, n = arguments.length; i < n; i++) {
            s = arguments[i];
            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
                t[p] = s[p];
        }
        return t;
    };
    return __assign.apply(this, arguments);
};

return [2, __assign(__assign({}, infoByBootloader), { binary: (0, exports.modifyFirmware)({ fw: fw, features: parsedFeatures }) })];
```

now it's ok to have it in 1 module (1 occurrence) but imagine to have it repeated 100x times for each module (check `packages/components/lib/...` files to see what i'm talking about)


if you build `suite-web` project with disabled minification and check for occurrences of `__assign` in build directory you will get 900+ results

Reply:
im going to correct myself :) i just find out that `@trezor/rollout` target is set to `es5` but to get rid of all unnecessary polyfills like `__assign`, `__awaiter` etc  should be set to `2017`

Reply:
@karliatto We had call with @szymonlesisz a we decided to do some changes in build `targed` mentioned above, so please get rid of this config option.